common/containers/Tabs/Swap/index.js

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
 import WantToSwapMy from './components/wantToSwapMy';
```

Comment:
Maybe think about naming based on what information user enters? eg coin type, address, value or amount.... 

Then you can use in / out so it's easier to follow thru the process. Incoming / outgoing coin, address, and so forth. 

Just idea